### PR TITLE
Update older version of LHCform features in lform schema

### PR DESCRIPTION
**Pull request by geovlazar**
Updated Changes in the Older versions features with following changes
**NihLhcForm**

- PATH_DELIMITER
- codeList
-  template
-  identifier
-  hasSavedData

**Interface FormTemplateOptions**

- showQuestionCode
- showCodingInstruction
- tabOnInputFieldsOnly
- showFormOptionPanelButton
- showItemOptionPanelButton
- useAnimation
- displayControl
- viewMode
- defaultAnswerLayout
- useTreeLineStyle
- columnHeaders

### CODE REVIEW

File: lform.ts [hunk 1]
```diff
@@ -220,3 +242,20 @@ export interface DatePeriod {
...
 }
+
+export interface CssExtensions {
+  extension?: [{
+    url?: string;
+    valueString?: string;
+  }];
+}
+
+export interface codeListOptions {
```
Comment (shah) [inline lform.ts:253]: This has improper capitalization - interface should be called `CodeListOptions` not codeListOptions.

File: lform.ts [hunk 2]
```diff
@@ -220,3 +242,20 @@ export interface DatePeriod {
...
+  code?: string;
+  display?: string;
+  system?: string;
+}
+
+export interface columnHeadOption {
```
Comment (shah) [inline lform.ts:259]: Please fix capitalization - make ColumnHeadOption not columnHeadOption.

**Review by shah — COMMENTED** [review]
Please make code formatting changes.

File: lform.ts (hunk 1 above)
Comment (geovlazar) [reply lform.ts]: Added the changes in the Interface

File: lform.ts (hunk 2 above)
Comment (geovlazar) [reply lform.ts]: Added the changes in the Interface

**Pull request merged**